apps/condo/domains/banking/schema/BankCostItem.test.js

```diff
@@ -0,0 +1,191 @@
+/**
+ * Generated by `createschema banking.BankCostItem 'name:Text;category:Relationship:BankCategory:SET_NULL'`
+ */
+
+const { makeLoggedInAdminClient, makeClient, UUID_RE, DATETIME_RE, expectToThrowGQLError, catchErrorFrom } = require('@open-condo/keystone/test.utils')
+
+const {
+    expectToThrowAuthenticationErrorToObj, expectToThrowAuthenticationErrorToObjects,
+    expectToThrowAccessDeniedErrorToObj,
+} = require('@open-condo/keystone/test.utils')
+
+const { makeClientWithNewRegisteredAndLoggedInUser } = require('@condo/domains/user/utils/testSchema')
+
+const { BankCostItem, createTestBankCostItem, updateTestBankCostItem } = require('@condo/domains/banking/utils/testSchema')
+const { createTestBankCategory } = require('../utils/testSchema')
+
+let admin
+let user
+let anonymous
+let category
+
+describe('BankCostItem', () => {
+    beforeAll(async () => {
+        admin = await makeLoggedInAdminClient()
```

Comment:
If there are some "business logic" related to restriction access to support, I think it would be better to add some test cases with that

Reply:
There is no additional business logic.
Items are readable for all, it will be hard-coded list, similar to categories.